lib/exactor/common.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ExActor.Common do
```

Comment:
Since you're introducing this module, then moving the code of `ExActor.Helper.init_generation_state` to the `__using__` might be nice. That way, we get to keep the common boilerplate in the same place.